### DISCUSSION TITLE
Is it possible to set app specific nginx properties in app.json or .env?

### DISCUSSION THREAD

**Discussion by frmdstryr** [dokku/dokku#6889] [Q&A]
The nginx client-max-body-size can't be set with --global, is it possible to set an app's nginx config from app.json somehow?

Thanks!

**Comment by josegonzalez** [comment]
Might be better to just make that settable as a global instead?

**Reply by frmdstryr** [reply]
Yeah, that would work for my current use case. 

At some point it'd be nice to have everything contained in the repo so git pushing auto-configures nginx for that app. If using app.json is not easy (or doesnt make sense to do), what about automatically copying files from an `nginx.conf.d` folder in the repo (if it exists) to the `/home/dokku/<app>/nginx.conf.d` folder or something like that?
